x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/utils/get_engines_display_text.test.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { getEnginesDisplayText } from './get_engines_display_text';
+import { IApiToken } from '../types';
+import { ADMIN, PRIVATE } from '../constants';
+
+const apiToken: IApiToken = {
+  name: '',
+  type: PRIVATE,
+  read: true,
+  write: true,
+  access_all_engines: true,
+  engines: ['engine1', 'engine2', 'engine3'],
+};
```

Comment:
We've used this in a couple test files now - I wonder if it's worth pulling out to a reusable const, perhaps in a `mocks.ts` file? I don't feel super strongly either way, I do think there's some nice value in having the example mock token immediately visible at the top of the test file vs having to check another file to see it.

Also sometimes I wonder why I post all these comments arguing against my own suggestions and if I should just delete them lol (sometimes I do delete them). I guess it's mostly just to throw things out there. Maybe it's so you all can see the numerous decisions I torture myself over every time I code. It's like livestreaming... but for code review. Good lord this analogy got away from me I'm going to stop now

Reply:
I wondered the same thing, but ended up keeping it inline since it's relatively small and I think I appreciated having it immediately referenceable when writing this test.

P.S. I don't mind seeing you type out your thought process, it gives me more context. :thu: